packages/indicators/src/__tests__/__integration/fixtures/eventCount/eventCountFixtures.ts

```diff
@@ -0,0 +1,144 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import {
+  ArraySurveyResponse,
+  ArrayTestCase,
+  arrayToSurveyResponse,
+  arrayToTestCase,
+} from '../helpers';
+import { entriesToEventCountIndicators } from './helpers';
+
+const ENTITIES = [
+  { code: 'AU', name: 'Australia', type: 'country' },
+  { code: 'NZ', name: 'New Zealand', type: 'country' },
+  { code: 'US', name: 'America', type: 'country' },
+];
+
+const SURVEYS = [
+  {
+    code: 'Covid_19_Cases',
+    questions: [
+      { code: 'Quarantine_Place', type: 'Radio' },
+      { code: 'Positive', type: 'Binary' },
+      { code: 'Age', type: 'Number' },
+    ],
+  },
+];
+
+const ARRAY_SURVEY_RESPONSES: ArraySurveyResponse[] = [
+  // AU
+  ['Covid_19_Cases', 'AU', '2020-03-01', { Quarantine_Place: 'Hotel', Positive: 'Yes', Age: '20' }],
+  ['Covid_19_Cases', 'AU', '2020-04-30', { Quarantine_Place: 'Home', Positive: 'No', Age: '25' }],
+  ['Covid_19_Cases', 'AU', '2020-05-29', { Quarantine_Place: 'Hotel', Positive: 'No', Age: '38' }],
+  ['Covid_19_Cases', 'AU', '2020-06-25', { Quarantine_Place: 'Home', Positive: 'Yes', Age: '40' }],
+  // NZ
+  ['Covid_19_Cases', 'NZ', '2020-03-01', { Quarantine_Place: 'Hotel', Positive: 'Yes', Age: '35' }],
+  ['Covid_19_Cases', 'NZ', '2020-03-30', { Quarantine_Place: 'Home', Positive: 'No', Age: '23' }],
+  ['Covid_19_Cases', 'NZ', '2020-04-30', { Quarantine_Place: 'Home', Age: '23' }], // Missing Quarantine_Place
+  // US
+  ['Covid_19_Cases', 'US', '2020-03-01', { Quarantine_Place: 'Hotel', Positive: 'Yes', Age: '22' }],
```

Comment:
The `US` data are not used, so we could delete them.

There is definitely a point to defining fixtures for multiple entities and verifying that only the specified entities are used. We already check this by having data for the same periods in `AU` and `NZ`.